plugins/score-card/src/components/ScoreCardTable/ScoreCardTable.test.tsx

```diff
@@ -110,6 +110,7 @@ describe('ScoreCard-TestWithData', () => {
   //   expect(container).toHaveTextContent('Custom title');
   // });
 
+  // TODO: Find why this test fail when entityRef is set on all.json scores
```

Comment:
I can't figure why this test is failing when one of the entity in `all.json` uses `entityRef`.
I'm not a react expert so I might need help here

Reply:
I will run it on my side

Reply:
It fails on `Error: Uncaught [Error: useHref() may be used only in the context of a <Router> component.]` and the source is `The above error occurred in the <Link> component` at `/home/jvilimek/backstage-plugins/plugins/score-card/src/components/ScoreCardTable/ScoreCardTable.tsx:51:30`

so the issue is somewhere here, as commenting out the test fails on assert (not error):
![image](https://user-images.githubusercontent.com/16001792/207381533-f13900e0-1c37-4312-a26a-b3b47f9e8e84.png)

Reply:
Finally fixed it using `MemoryRouter`

Reply:
Nice work!